search/query/conjunction.go

```diff
@@ -28,46 +28,74 @@ import (
 
 // ConjuctionQuery finds documents which match at least one of the given queries.
 type ConjuctionQuery struct {
-	Queries []search.Query
+	Queries   []search.Query
+	Negations []search.Query
```

Comment:
cant this be in the `Queries` field? I ask because it'll be annoying to add this to the RPC layer

Reply:
Err, crap, yea ideally this would be an internal detail, let's discuss tomorrow